build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
         maven { url 'https://plugins.gradle.org/m2/' }
```

Comment:
let's update the depot version 0.8.0 and also firehose version to 0.10.0

Reply:
done